### PR TITLE
Add `require-top-level-describe` rule

### LINKED ISSUE #84: require-top-level-describe
https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/require-top-level-describe.md

### PR DESCRIPTION
**Pull request by mskelton**
Fixes #84

### CODE REVIEW

**Comment by mxschmitt** [comment]
Not sure if we should recommend that by default. I‘d vote against.

**Review by mxschmitt — APPROVED** [review]

**Comment by mskelton** [comment]
> Not sure if we should recommend that by default. I‘d vote against.


Totally agree, this PR adds the rule, but not enabled by default!

File: src/rules/require-top-level-describe.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        }
+      },
+    };
+  },
+  meta: {
+    docs: {
+      category: 'Best Practices',
+      description:
+        'Require test cases and hooks to be inside a `test.describe` block',
+      recommended: true,
```
Comment (mxschmitt) [inline src/rules/require-top-level-describe.ts:52]: Looks recommended?

Comment (mskelton) [reply src/rules/require-top-level-describe.ts]: Ah good catch.  Updated this to false and fixed a few other rules which were marked incorrectly

**Pull request merged**